### PR TITLE
Add test for KcAdm status change

### LINKED ISSUE #33240: Add tests for the new kcadm `--status` option
### Description

Some tests should be added for the new option that was introduced in https://github.com/keycloak/keycloak/pull/32883.

### PR DESCRIPTION
**Pull request by keshavprashantdeshpande**
Closes #33240

### CODE REVIEW

File: testsuite/integration-arquillian/tests/base/src/test/java/org/keycloak/testsuite/cli/admin/KcAdmTest.java [hunk 1]
```diff
@@ -684,4 +684,20 @@ public void testEnvPasswordWithRegularCommand() {
         assertExitCodeAndStreamSizes(exec, 0, 1, 0);
     }
 
+    @Test
+    public void testStatusOptionForConfigCred() {
+        KcAdmExec exec = execute("config credentials --status");
+        assertExitCodeAndStreamSizes(exec, 0, 1, 0);
+        Assert.assertEquals("You are not logged in\n", exec.stdoutString());
```
Comment (shawkins) [inline testsuite/integration-arquillian/tests/base/src/test/java/org/keycloak/testsuite/cli/admin/KcAdmTest.java:691]: @keshavprashantdeshpande these tests have a user logged in by default, so this first assertion is failing.

Comment (keshavprashantdeshpande) [reply testsuite/integration-arquillian/tests/base/src/test/java/org/keycloak/testsuite/cli/admin/KcAdmTest.java]: @shawkins I have removed the assert and checks seem to work again.  Can you please review the changes?

**Review by mabartos — APPROVED** [review]
@keshavprashantdeshpande Thanks for the PR!

@shawkins Do you wanna check it as well?

**Review by shawkins — APPROVED** [review]
Thank you @keshavprashantdeshpande

**Pull request merged**